### ISSUE TITLE
Swift Package Manager installation is failing.

### ISSUE THREAD

**Issue by zorn** [bignerdranch/Freddy#224]
I tried to add Freddy to a simple Hello World Swift Package Manager app (first time playing with it). My `Package.swift` file:

````
import PackageDescription

let package = Package(
    name: "Hello",
    targets: [],
	dependencies: [
		.Package(url: "https://github.com/bignerdranch/Freddy.git", majorVersion: 3),
    ]
)
````

When I ran `swift build` I got:

````
$ swift build
Cloning https://github.com/bignerdranch/Freddy.git
HEAD is now at c2780e5 Merge pull request #218 from bignerdranch/swift3
Resolved version: 3.0.0
Empty manifest file is not supported anymore. Use `swift package init` to autogenerate.
error: The package at `/Users/zorn/Desktop/Hello/Packages/Freddy.git' has no Package.swift for the specific version: 3.0.0
````

Looks like we need to populate `Package.swift` with info now.

**Comment by zorn** [comment]
Seems like this is fixed in PR #223. Sadly I don't see a way to request a specific branch in an actual dependency so I don't know how to help verify this fixes the issue without merging into `master`.

<https://github.com/apple/swift-package-manager/blob/master/Documentation/Reference.md#package-dependency>

**Comment by mdmathias** [comment]
For the sake of documentation:

There is a now a [release tagged as 3.0.1](https://github.com/bignerdranch/Freddy/releases/tag/3.0.1). Your `Package.swift` should like this to make use of it:

```swift
import PackageDescription

let package = Package(
    name: "Hello",
    dependencies: [
        .Package(url: "https://github.com/bignerdranch/Freddy.git", Version(3,0,1)),
    ]
)
```
